src/components/table/row/index.tsx

```diff
@@ -7,36 +7,24 @@ import { Component, Element, Event, EventEmitter, h, Host, Listen, Prop, State,
 })
 export class TableRow {
 	@Element() element: HTMLSmoothlyTableRowElement
-	expansionElement?: HTMLTableRowElement
-	@Event() expansionOpen: EventEmitter<HTMLElement>
-	@Prop() align: "left" | "center" | "right" = "left"
-	@Prop({ mutable: true, reflect: true }) open: boolean
-	@State() beginOpen: boolean
-	@Watch("open")
-	openChanged(value: boolean) {
-		if (this.expansionElement)
-			this.element.after(this.expansionElement)
+	expansions: HTMLSmoothlyTableExpandableCellElement[] = []
+	@Listen("expansionLoaded")
+	onExpansionLoaded(event: CustomEvent<void>) {
```

Comment:
Event names should be present form e.g. `click`, `hover`, `scroll`.
I know you can examples of past-form on event names in smoothly (made by me). But those are mistakes and should be corrected.
```suggestion
	@Listen("expansionLoad")
	onExpansionLoad(event: CustomEvent<void>) {
```